static/src/javascripts/lib/__mocks__/fastdom.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+/* eslint-disable */
```

Comment:
Are we disabling ESLint because of unused vars? Maybe we should be explicit here, to help future archaeologists understand why we're doing this

```js
/* eslint-disable no-unused-vars */
```